lib/src/components/NxCheckbox/NxCheckbox.scss

```
@@ -21,24 +25,38 @@ $-width: 16px;
     @include nx-focus-border-helpers.radio-checkbox-focus-border($-height, $-width);
 
     border-radius: 4px;
-    left: -5px;
-    top: -5px;
+    left: calc(var(--nx-focus-outline-offset) + var(--nx-focus-outline-width));
+    top: -4px;
   }
 
   .fa-check {
-    color: var(--nx-swatch-white);
+    color: var(--nx-color-component-background);
     font-size: var(--nx-font-size-s);
-    vertical-align: 4px;
   }
 }
 
-.nx-checkbox__input:checked:disabled {
-  + .nx-checkbox__box {
-    background-color:var(--nx-color-interactive-background-disabled);
-    border-color: var(--nx-color-interactive-border-disabled);
+.nx-checkbox__input {
+  &:checked {
+    + .nx-checkbox__box {
+      @include nx-radio-checkbox-helpers.inset-box-shadow(var(--nx-color-radio-checkbox-selected));
```

This block could equivalently be
```suggestion
      background-color: var(--nx-color-radio-checkbox-selected);
      
      &, &:hover {
        @include nx-radio-checkbox-helpers.inset-box-shadow(var(--nx-color-radio-checkbox-selected));
      }
```

---

> This block could equivalently be
> 
> ```
>       background-color: var(--nx-color-radio-checkbox-selected);
>       
>       &, &:hover {
>         @include nx-radio-checkbox-helpers.inset-box-shadow(var(--nx-color-radio-checkbox-selected));
>       }
> ```

Yup, fixed: a81ecdf26ef5489af7702df498b135c0b45e9e8a